### PR TITLE
Rewrite Storyscript Intro

### PR DESCRIPTION
**Pull request by stevepeak**
Feedback welcome ❤️ 

Based on the following feedback:
- **Show me the code!** People want to see code ASAP
- **Why?** Should you care?
- Identify key take-aways
- Classic: problem, solution, benefits, uses

My next major work is going into more details on **HOW** this works. Since it's become the next obvious question. But this page is more reserved for **WHY** we are here.

### CODE REVIEW

File: support/README.md [hunk 1]
```diff
@@ -2,27 +2,27 @@
 
 <!--
 
-| Community Forum |  Asyncy Cloud Status |
+| Community Forum |  Storyscript Cloud Status |
 | :-----------: | :-----------------: |
 | QA, community support and feature requests | System health, metrics and issues | Issue tracking |
-| [forum.asyncy.com](https://forum.asyncy.com) | [status.asyncy.com](https://status.asyncy.com) |
+| [forum.storyscript.io](https://forum.storyscript.io) | [status.storyscript.io](https://status.storyscript.io) |
```
Comment (judepereira) [inline support/README.md]: we don't have forums - please remove.

Comment (judepereira) [reply support/README.md]: Instead, best way is support@storyscript.io

File: support/README.md [hunk 2]
```diff
@@ -2,27 +2,27 @@
...
-Feedback is welcome and much appreciated. [Submit anonymous feedback](https://asyncy.click/feedback) on the Asyncy Platform.
+Feedback is welcome and much appreciated. [Submit anonymous feedback](https://asyncy.click/feedback) on the Storyscript Platform.
 
 ## Contribute
 
 We look forward to your contributions to our Open Source platform. :heart:
 
 - For first-time contributors we suggest checking out issues tagged with [{good first issue}](https://github.com/issues?q=is:open+is:issue+archived:false+user:storyscript+label:%22good+first+issue%22) or  [{help wanted}](https://github.com/issues?q=is:open+is:issue+archived:false+user:storyscript+label:%22help+wanted%22).
-- The [community forum](https://forum.asyncy.com) has questions that you can help help answer too.
+<!-- - The [community forum](https://forum.storyscript.io) has questions that you can help help answer too. -->
```
Comment (judepereira) [inline support/README.md]: remove forum.

File: support/README.md [hunk 3]
```diff
@@ -2,27 +2,27 @@
...
-| Community Forum |  Asyncy Cloud Status |
+| Community Forum |  Storyscript Cloud Status |
 | :-----------: | :-----------------: |
 | QA, community support and feature requests | System health, metrics and issues | Issue tracking |
-| [forum.asyncy.com](https://forum.asyncy.com) | [status.asyncy.com](https://status.asyncy.com) |
+| [forum.storyscript.io](https://forum.storyscript.io) | [status.storyscript.io](https://status.storyscript.io) |
 
 ::: tip Need help?
-Please search our [Community Forum](https://forum.asyncy.com) for help or ask a new question.
+Please search our [Community Forum](https://forum.storyscript.io) for help or ask a new question.
```
Comment (judepereira) [inline support/README.md]: remove forum.

File: storyscript/intro/README.md [hunk 4]
```diff
@@ -7,67 +7,79 @@ next: /storyscript/writing/
...
-2. The second line represents taking action on the incoming event data, harnessing another microservice.
+### The next abstraction: Top-level Programming
 
+**Low-level languages** empowered people to create extraordinary processes in a form that was significantly more readable than machine-code, yet left us with managing memory.
 
-```coffeescript
-when service action event key:value as event  # Event-driven microservice
-  res = service action key:value              # HTTP/RPC microservice
-  result = someFunction(key:value)            # Any GPPL/Storyscript Function
-  someString.upper()                          # A type transformation
```
Comment (wilzbach) [inline storyscript/intro/README.md]: Maybe use `result` here, s.t. we use a defined variable?

File: storyscript/intro/README.md [hunk 5]
```diff
@@ -7,67 +7,79 @@ next: /storyscript/writing/
...
+✨Storyscript is the cloud-native, polyglot programming language that seamlessly moves data between microservices and functions with zero-devops deployments into Kubernetes. Stay focused on what matters most: business-logic. *Everything else is noise.* ✨
 
-The design of Storyscript is to **move data**, with no boilerplate code, in a declarative and intuitive way.
+Let's start with the code. The snippet below shows the key features of Storyscript.
 
-::: tip Inspiration behind Storyscript
-Storyscript is inspired by many popular languages to be as natural and intuitive as possible. It is declarative, strong-typed, static-typed and focused on top-level data-flow.
+```coffeescript
+when service action eventName as event        # An event-driven microservice
+  result = aPythonFunction(foo:event['name']) # Call a function in any language
```
Comment (wilzbach) [inline storyscript/intro/README.md]: Double quotes.

File: storyscript/intro/README.md [hunk 6]
```diff
@@ -7,67 +7,79 @@ next: /storyscript/writing/
...
+Ideally, development would be focused exclusively on necessary complexity but the industry is far from that. Today, one could argue that most our time is spend in unnecessary and accidental complexities. Storyscript aims to significantly reduce development complexities resulting in a steep improvement in productivity.
 
-Many think of Storyscript as glue code for microservices, flow-code, or pure business-logic programming. We call this **top-level programming**. This diagram shows how the language focuses on the most important aspect of application development: the business-logic.
+### Staying Super-DRY
 
+Development is soaking wet in technical debt 🌊. Yes, that's catchy, but very true. As an industry we need to ask ourselves: *why we are creating libraries for every language?* *Why are we responsible for soo much unnecessary complexity?* *When is the promise of Lego-style development coming true?*
```
Comment (wilzbach) [inline storyscript/intro/README.md]: "Why are we" also for the first question?

File: support/README.md (hunk 1 above)
Comment (stevepeak) [reply support/README.md]: Yes, this is commented out atm.

File: storyscript/intro/README.md (hunk 5 above)
Comment (stevepeak) [reply storyscript/intro/README.md]: Nice catch 👍

**Pull request merged**